mne/io/edf/edf.py

```diff
@@ -1922,7 +1922,7 @@ def _read_annotations_edf(annotations, encoding="utf8"):
     if isinstance(annotations, str):
         with open(annotations, "rb") as annot_file:
             triggers = re.findall(pat.encode(), annot_file.read())
```

Comment:
Do we also need to pass it to `pat.encode(encoding=encoding)` as well?

Reply:
I think not as pat was defined with utf-8 encoding (i.e. "\x00" is the UTF-8 encoding for U+0000)